examples/summarization/README.md

```diff
@@ -225,3 +225,28 @@ python run_summarization.py \
     --ignore_pad_token_for_loss False \
     --pad_to_max_length
 ```
+
+You can run inference with BART on the CNN-DailyMail dataset on 1 Gaudi card with the following command:
+```bash
+python run_summarization.py \
+    --model_name_or_path facebook/bart-large-cnn \
+    --do_predict \
+    --dataset_name cnn_dailymail \
+    --dataset_config "3.0.0" \
+    --output_dir /tmp/tst-summarization \
+    --per_device_eval_batch_size 2 \
+    --overwrite_output_dir \
+    --predict_with_generate \
+    --use_habana \
+    --use_lazy_mode \
+    --use_hpu_graphs_for_inference \
+    --gaudi_config_name Habana/t5 \
+    --ignore_pad_token_for_loss False \
+    --pad_to_max_length \
+    --num_beams 1 \
+    --use_token_idx
```

Comment:
```suggestion
    --num_beams 1
```
`token_idx` should be automatically used after adding BART to this list of optimized models: https://github.com/huggingface/optimum-habana/blob/7204bbbb1e28e11c3b99ff6e3b5a2bc9930886dc/optimum/habana/transformers/generation/utils.py#L60

Reply:
Done